app/client/ui/AdminPanel.ts

```diff
@@ -649,3 +649,19 @@ export const cssLabel = styled('div', `
   text-align: right;
   padding-right: 5px;
 `);
+
+
+/**
+ * Make a long code to use in the example, so that if people copy
+ * and paste it lazily, they end up decently secure, or at least a
+ * lot more secure than a key like "REPLACE_WITH_YOUR_SECRET"
+ */
+function _longCodeForExample() {
+  // Crypto in insecure contexts doesn't have randomUUID
+  if (window.isSecureContext) {
+    return 'example-a' + window.crypto.randomUUID();
+  }
+  return 'example-b' + 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'.replace(/[x]/g, () => {
```

Comment:
Why the square brackets? Seems unusual for a character class with a single member.